mix.exs

```diff
@@ -14,6 +14,7 @@ defmodule Penelope.Mixfile do
       description: description(),
       deps: deps(),
       package: package(),
+      source_url: "https://github.com/pylon/penelope",
```

Comment:
What is this used for?

Reply:
I actually put this in awhile ago and haven't had a PR opportunity to include it until now. I had to look it up again—it's used by ExDoc for formatting the project description; it's in the docs [here](https://hex.pm/docs/publish).

Reply:
Ah, I didn't know about that one. Weird that it's used in the project and the package config.